.gitignore

```diff
@@ -1,2 +1,4 @@
 /node_modules
 /*.log
+.idea/
+*.iml
```

Comment:
It's not hurting anything, so don't worry about it. I generally use that canned comment for when people add `.DS_Store` to my `.gitignore` ;)